process/block/baseProcess.go

```diff
@@ -2405,3 +2405,11 @@ func (bp *baseProcessor) checkReceivedProofIfAttestingIsNeeded(proof data.Header
 		bp.chRcvAllHdrs <- true
 	}
 }
+
+func (bp *baseProcessor) getMaxRoundsWithoutBlockReceived(round uint64) uint64 {
```

Comment:
perhaps extract this method to common? same for the constants

Reply:
i would handle refactor, together with the other TODOs in a future PR, when we'll add config per epoch